invenio_app_ils/ui/src/invenio_app_ils/pages/backoffice/DocumentsSearch/DocumentsSearch.js

```diff
@@ -160,7 +133,23 @@ export class DocumentsSearch extends Component {
         <Aggregator title={agg.title} field={agg.field} />
       </div>
     ));
-    return <div>{components}</div>;
+    return (
+      <>
+        <NewButton
+          fluid
+          text={'New document'}
+          clickHandler={goToHandler(BackOfficeRoutes.documentCreate)}
```

Comment:
This one cannot be refactored with your link approach?

Reply:
Because there are several places that we use the clickHandlers instead of LInks, thats why I moved them to a different pr check the comment above.